src/vueWrapper.ts

```diff
@@ -159,8 +159,11 @@ export class VueWrapper<T extends ComponentPublicInstance>
   ): VueWrapper<T> {
     if (typeof selector === 'object' && 'ref' in selector) {
       const result = this.vm.$refs[selector.ref]
-      if (result) {
+      console.log(result)
```

Comment:
```suggestion
```

Reply:
@afontcu Thank you! One day I will remember to have another SPECIFIC look for console.logs and debugger :)